### PR TITLE
Add GDAL_HFA_OVR_BLOCKSIZE

### PR DESCRIPTION
**Pull request by drons**
Add GDAL_HFA_OVR_BLOCKSIZE configuration option to control HFA overviews block size

### CODE REVIEW

**Comment by drons** [comment]
@rouault 
I think that it is necessary to move out the function IsPowerOfTwo from the files hfaband.cpp and geotiff.cpp in a common place, but I do not know where. Any suggestions?

**Comment by rouault** [comment]
> I think that it is necessary to move out the function IsPowerOfTwo from the files hfaband.cpp and geotiff.cpp in a common place, but I do not know where. Any suggestions?

Perhaps in port/cpl_conv.cpp, with a CPL prefix in the name. There's another occurence in ogr/ogrsf_frmts/sqlite/rasterlite2.cpp

Oh, and unrelated, but as you are a regular contributor, I feel bad not to even know your full name but just your GH pseudo

File: gdal/port/cpl_conv.h [hunk 1]
```diff
@@ -281,6 +281,11 @@ char* CPLsetlocale (int category, const char* locale);
 void CPLCleanupSetlocaleMutex(void);
 /*! @endcond */
 
+/* -------------------------------------------------------------------- */
+/*                          CPLIsPowerOfTwo()                           */
+/* -------------------------------------------------------------------- */
+bool CPL_DLL CPLIsPowerOfTwo( unsigned int i );
```
Comment (rouault) [inline gdal/port/cpl_conv.h]: We try to stick to C89, so either return int, or put it in the C++ specific part of the header file (which is OK as the intended usage is rather internal)

**Comment by rouault** [comment]
Doxygen warning: warning: Member CPLIsPowerOfTwo(unsigned int i) (function) of file cpl_conv.h is not documented.

**Comment by rouault** [comment]
Committed in r41262 + r41263. Thanks

**Comment by drons** [comment]
Thanks!

**Pull request closed without merging**